docs/content/stable/manage/upgrade-deployment.md

```diff
@@ -75,6 +75,47 @@ Use the following procedure to upgrade a YB-TServer:
 
 1. Pause for approximately 60 seconds before upgrading the next YB-TServer.
 
+## Promote AutoFlags
+A lot of new Yugabyte features require changes to the format of data that is sent over the wire or stored on disk. During the upgrade process, these features have to be turned off to prevent sending the new data formats to nodes that are still running the older version. Once all Yugabyte processes have been upgraded to the new version, these features can be safely enabled.
+
+[AutoFlags](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/auto_flags.md) help facilitate this transition without requiring you to worry about identifying these features, finding their corresponding flags, or determining the values to set them to. All new AutoFlags can be easily promoted to their desired target value using a simple command.
+
+Use the [yb-admin](../../admin/yb-admin/) utility to promote the new AutoFlags, as follows:
+
+```sh
+./bin/yb-admin \
+    -master_addresses <master-addresses> \
+    promote_auto_flags
+```
+
+The promotion of AutoFlags is an online operation that does not require stopping a running cluster or any process restarts. It is also an idempotent process, meaning it can be run multiple times without any side effects.
+
+Please note that it may take up to twice the value of `FLAGS_heartbeat_interval_ms` in milliseconds for the new AutoFlags to be fully propagated to all processes in the cluster.
```

Comment:
```suggestion
Note that it may take up to twice the value of `FLAGS_heartbeat_interval_ms` in milliseconds for the new AutoFlags to be fully propagated to all processes in the cluster.
```